src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -75,6 +77,18 @@ const createUiSchemaForString = (ionFormField, remediationForm, transformedResp,
     }
   }
 
+  if(ionFormField.name === 'userProfile.countryCode'){
+    uiSchema.type = 'select';
+    uiSchema.wide = true;
+    uiSchema.options = CountryUtil.getCountries();
+  }
+
+  if(ionFormField.name === 'userProfile.timezone'){
+    uiSchema.type = 'select';
+    uiSchema.wide = true;
+    uiSchema.options = TimeZone;
+  }
+
```

Comment:
Can we maybe add some tests to make sure both drop-downs get populated properly?

Reply:
good question. I have not found one yet